lib/msf/core/exploit/http/joomla/base.rb

```diff
@@ -0,0 +1,30 @@
+# -*- coding: binary -*-
+
+module Msf::Exploit::Remote::HTTP::Joomla::Base
+
+  # Checks if Joomla is up and running.
+  #
+  # @return [TrueClass] Joomla is up and running.
+  # @return [FalseClass] Joomla is not up.
+  def joomla_and_online?
+    # Possible paths that we might see the generator tag.
+    paths = [ '/', '/administrator' ]
```

Comment:
This should be `/administrator/` otherwise strange things and redirects happen o_O